### DISCUSSION TITLE
Erro Ficheiro Saft

### DISCUSSION THREAD

**Discussion by RodolfomsFrancisco** [assoft-portugal/SAF-T-AO#162] [Q&A] [answered]
Bom dia a todos estou tendo um erro muito estranho e não consigo resolver porfavor alguem pode  me dar uma dica de como resolver 

esté é a mensagem que recebo!...

Erro no ficheiro, AuditFile.SourceDocuments.SalesInvoices.1Invoice (FR 2021/1) .DocumentTotals.GrossTotal é maior que o documento de origem.
--
Erro no ficheiro, AuditFile.SourceDocuments.SalesInvoices.2Invoice (FR 2021/2) .DocumentTotals.GrossTotal é maior que o documento de origem.
Erro no ficheiro, AuditFile.SourceDocuments.SalesInvoices.3Invoice (FR 2021/3) .DocumentTotals.GrossTotal é maior que o documento de origem.

**Comment by cryptolopes** [comment, marked as answer]
Está relacionado com um [problema na plataforma](https://t.me/saftao/5243) da AGT. Deve ser ignorado. De qualquer das formas, comporta-se como um **alerta** e não como um erro, conforme o feedback que temos tido sobre o assunto.

**Reply by RodolfomsFrancisco** [reply]
muito obrigado pela resposta caríssimo fico mais descansado !...
